tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -311,6 +311,43 @@ public void Builds_Net5MvcApp_UsingNet5_DotNetSdkVersion()
                 result.GetDebugInfo());
         }
 
+        // This test is necessary once .NET 6 preview 5 come out.
+        [Fact(Skip = "Temporarily skip for issue https://github.com/dotnet/sdk/issues/17865")]
+        public void Builds_Net6BlazorWasmApp_RunsAOTCompilationInstallCommands()
```

Comment:
we should have this PR when we support preview 5 else we are potentially adding dead code or breaking current .net 6 blazor apps